src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,127 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { changeTodo } from '../../api/todos';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  temp?: boolean,
+  deleteTodo?: (id: number) => void,
+  updateTodos?: (type?: string) => void
+};
+
+export const Item: React.FC<Props> = ({
+  todo, temp, deleteTodo, updateTodos,
+}) => {
+  const [loader, setLoader] = useState(false);
+  const [change, setChange] = useState(false);
+  const [title, setTitle] = useState(todo.title);
+
+  const activeLoader = () => {
+    if (deleteTodo) {
+      deleteTodo(todo.id);
+      setLoader(true);
+    }
+  };
+
+  const changeTodoCompleteHandler = (changeName:boolean) => {
```

Comment:
It is better to name like this if you use a boolean
```suggestion
  const changeTodoCompleteHandler = (isChangedName:boolean) => {
```